gwcs/conftest.py

```diff
@@ -1,28 +1,8 @@
-import os
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
 
-# this contains imports plugins that configure py.test for astropy tests.
-# by importing them here in conftest.py they are discoverable by py.test
-# no matter how it is invoked within the source tree.
+# This contains plugins that configure py.test for asdf tests.
 
-from astropy.tests.pytest_plugins import *
-
-# Uncomment the following line to treat all DeprecationWarnings as
-# exceptions
-enable_deprecations_as_exceptions()
-
-try:
-    PYTEST_HEADER_MODULES['Astropy'] = 'astropy'
-    PYTEST_HEADER_MODULES['asdf'] = 'asdf'
-    del PYTEST_HEADER_MODULES['h5py']
-except (NameError, KeyError):
-    pass
-
-# This is to figure out the affiliated package version, rather than
-# using Astropy's
-from . import version
-
-try:
-    packagename = os.path.basename(os.path.dirname(__file__))
-    TESTED_VERSIONS[packagename] = version.version
-except NameError:   # Needed to support Astropy <= 1.0.0
-    pass
```

Comment:
Is there a reason for removing all of this? It shouldn't be necessary to get the asdf pytest plugin working.

Reply:
No, no reason. I will restore it.